src/assets/sass/variables/colors.scss

```
@@ -67,6 +67,9 @@ $main-value-summary-text: #58b1da;
 $green: #059669;
 $orange: #d97706;
 
+$expense-maroon: #8b4459;
```

Stylelint (17.13.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)